updatesnap/unittests.py

```diff
@@ -8,7 +8,7 @@
 import sys
 import yaml
 from SnapModule.snapmodule import Snapcraft
-from SnapModule.snapmodule import ManageYAML
+from SnapModule.manageYAML import ManageYAML
```

Comment:
Can you add some unitary tests that check this new option?

Reply:
@sergio-costas, updated unittests for snap version automation